lunatrace/README_DEV_SETUP.md

```diff
@@ -132,13 +131,12 @@ will ensure that your Kratos version is always in sync with the Docker version._
 
 #### Hasura (second):
 
-Run...
+From the `bsl/hasura` folder run...
 
 ```sh
-cd bsl/hasura
+hasura migrate apply
```

Comment:
same about revert, putting the cd in the code block here allows you to click to run it.